### PR TITLE
Feat: JRD - Filter the new Personal file using updated values in Appointment

### PR DESCRIPTION
**Pull request by souravroy2021**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-2845

### Change description ###

JRD : Filter the new Personal file using updated values in Appointment Type field.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by shreedhar-hmcts — APPROVED** [review]

**Review by kavithagade12 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/juddata/validators/AppointmentValidator.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.apache.commons.lang3.StringUtils;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.Set;
+
+public class AppointmentValidator implements ConstraintValidator<Appointment, String> {
+
+    private static final String VALUE1 = "CRTS TRIB - RS Admin User";
+    private static final String VALUE2 = "MAGS - AC Admin User";
```
Comment (abhijitdiwan) [inline src/main/java/uk/gov/hmcts/reform/juddata/validators/AppointmentValidator.java:12]: Can you please move these in constant class

File: src/main/java/uk/gov/hmcts/reform/juddata/validators/Appointment.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Target({METHOD, FIELD, PARAMETER})
```
Comment (abhijitdiwan) [inline src/main/java/uk/gov/hmcts/reform/juddata/validators/Appointment.java:14]: Trivial comment but ideally @Target({FIELD}) would suffice

File: src/main/java/uk/gov/hmcts/reform/juddata/validators/AppointmentValidator.java (hunk 1 above)
Comment (souravroy2021) [reply src/main/java/uk/gov/hmcts/reform/juddata/validators/AppointmentValidator.java:12]: These constants are not reusable and strongly tied to existing class.

According to Joshua Bloch in his book Effective Java (Item 19 in 2nd Ed):

If the constants are strongly tied to an existing class or interface, you should add them to the class or interface.

**Comment by souravroy2021** [comment]
Need to create a new PR in order to merge into master.

**Pull request closed without merging**